packages/kbn-optimizer/README.md

```diff
@@ -0,0 +1,111 @@
+# @kbn/optimizer
+
+`@kbn/optimizer` is a package for building new platform UI plugins (and hopefully more soon).
+
+New Platform plugins with `"ui": true` in their `kibana.json` file will have their `public/index.ts` file (and all of its dependencies) bundled into the `target/public` directory of the plugin. The build output does not need to be updated when other plugins are updated and is included in the distributable without requiring that we ship `@kbn/optimizer` 🎉.
+
+## Webpack config
+
+The [Webpack config][WebpackConfig] is designed to provide the majority of what was available in the legacy optimizer and is the same for all plugins to promote consistency and keep things sane for the operations team. It has support for JS/TS built with babel, url imports of image and font files, and support for importing `scss` and `css` files. SCSS is pre-processed by [postcss][PostCss], built for both light and dark mode and injected automatically into the page when the parent module is loaded (page reloads are still required for switching between light/dark mode). CSS is injected into the DOM as it is written on disk when the parent module is loaded (no postcss support).
+
+Source maps are enabled except when building the distributable. They show the code actually being executed by the browser to strike a balance between debuggability and performance. They are not configurable at this time but will be configurable once we have a developer configuration solution that doesn't rely on the server (see [#55656](https://github.com/elastic/kibana/issues/55656)).
```

Comment:
Wondering if this will be an issue for folks.

Reply:
Definitely possible, we can adapt if people require configurability. I think people will find that I have found, which is that the inspector is a lot more useful when it shows the code that's actually executing. Especially when it comes to the name of local variables and stepping through lines.